app/scripts/modules/core/src/artifact/expectedArtifact.service.ts

```diff
@@ -31,4 +32,29 @@ export class ExpectedArtifactService {
         .filter(v => v)
         .reduce((array, value) => array.concat(value), []);
   }
+
+  public static createEmptyArtifact(kind: string): IExpectedArtifact {
+    return {
+      id: UUIDGenerator.generateUuid(),
+      usePriorArtifact: false,
+      useDefaultArtifact: false,
+      matchArtifact: {
+        id: UUIDGenerator.generateUuid(),
```

Comment:
I am unclear if the ID for an artifact is intended to equal the id of its ExpectedArtifact so I've made them all different for now.

Reply:
I think the intent is for them to be different, so this looks good